src/main/java/com/dropwizard/example/h2/resources/PersonResource.java

```diff
@@ -0,0 +1,97 @@
+package com.dropwizard.example.h2.resources;
```

Comment:
Person and PersonResource are not part of h2 at all, thus please place it in proper package. Like
> com.dropwizard.example.model
> com.dropwizard.example.resource